packages/testing/src/test/create-test-disposables.unit.ts

```diff
@@ -0,0 +1,51 @@
+import { expect } from 'chai';
+import { createTestDisposables } from '../create-test-disposables';
+/**
+ * NOTE: these tests assume to run in order! and order of creation matter!.
+ * this is because we test the integration itself to mocha, during mocha tests.
+ */
+describe('createTestDisposables', () => {
+    const disposed = [] as string[];
+    // before we create the disposables
+    afterEach(() => {
+        expect(disposed.length, 'nothing disposed yet!').to.eql(0);
+    });
+    // create the disposables
+    const td = createTestDisposables();
+    // after each test we expect one handler to be disposed
+    afterEach(() => {
+        expect(disposed.length, 'one handler after the each test').to.eql(1);
+        disposed.length = 0;
+    });
+    it('test1', () => {
+        td.add('test1', () => disposed.push('test1'));
+    });
+    it('test2', () => {
+        td.add('test2', () => disposed.push('test2'));
+    });
+});
+
+describe('createTestDisposables', () => {
```

Comment:
set more informative describe names

Reply:
What info is missing?

Reply:
Two describes with the same name, one should be maybe `createTestDisposables - afterEach (default)` and the other can be `createTestDisposables - after`

Reply:
you could also wrap them both in a single top level describe